recipes/libgeoip/build.bat

```diff
@@ -0,0 +1,6 @@
+mkdir build
+cd build
+
+nmake /f Makefile.vc
```

Comment:
Instead of editing you can also set `GEOIPDATADIR` and `INSTDIR` on the command line